### ISSUE TITLE
Unable to install ntp on Cooker, missing dependencies

### ISSUE THREAD

**Issue by edwardp3** [OpenMandrivaAssociation/distribution#2913]
**OpenMandriva version**:
Cooker

**Describe the bug**:
Attempted to install ntp (chrony is unsupported) and it would not install, as nothing provided missing dependencies.

**Steps to reproduce**:
Attempt to install ntp, dnf would not install due to missing dependencies.

**Logs and screenshots if relevant**

```
Last metadata expiration check: 0:27:01 ago on Mon 08 May 2023 05:27:07 PM EDT.
Error: 
 Problem: conflicting requests
  - nothing provides libnetsnmp.so.35()(64bit) needed by ntp-4.2.8p13-1.x86_64
  - nothing provides libnetsnmpagent.so.35()(64bit) needed by ntp-4.2.8p13-1.x86_64
(try to add '--skip-broken' to skip uninstallable packages)
```

**Comment by edwardp3** [comment]
Learned OM is using systemd-timesyncd, so please feel free to close this, if necessary.

**Comment by berolinux** [comment]
ntp is still supposed to work even if it's not the default (it can still be useful for e.g. running an ntp server).

**Comment by edwardp3** [comment]
I just edited my comment to reflect _my system_. In any event, I'm glad I reported the bug so the installation can be remedied.

**Comment by berolinux** [comment]
Fixed in cooker (4.2.8p15-1)
